CRM/Report/Form/Contribute/Bookkeeping.php

```diff
@@ -3,7 +3,7 @@
  +--------------------------------------------------------------------+
  | CiviCRM version 4.7                                                |
  +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC (c) 2004-2015                                |
+ | Copyright CiviCRM LLC (c) 2004-2016                                |
```

Comment:
Are you able to remove these lines from your change? They seem correct & probably need changing but they should be done en masse. You might want to log a JIRA to remind someone to change them...

@JoeMurray I don't really work with batches - can you eyeball the join below - or tag someone else who can?

I didn't expect batch to join like this

LEFT JOIN civicrm_entity_batch ent_batch
+                    ON  {$this->_aliases['civicrm_financial_trxn']}.id = ent_batch.entity_id AND ent_batch.entity_table = 'civicrm_financial_trxn'

but that doesn't mean it doesn't!